build/Targets/GenerateCompilerExecutableBindingRedirects.targets

```diff
@@ -5,7 +5,7 @@
   <PropertyGroup>
     <AutoGenerateBindingRedirects>true</AutoGenerateBindingRedirects>
   </PropertyGroup>
-
+  
```

Comment:
@Pilchie VS has been doing this for C# files for a very long time IIRC. Not other files though, which would be nice. No reason not to. (self note: I hate how GH hides comments...)